packages/cozy-procedures/src/components/PersonalDataForm.jsx

```diff
@@ -25,10 +26,11 @@ const widgets = {
 
 class PersonalDataForm extends React.Component {
   render() {
-    const { formData, updateFormData, router } = this.props
+    const { formData, updateFormData, router, t } = this.props
     return (
       <div className="u-ml-2">
-        <h2 className="u-title-h2">Personal data</h2>
+        <h2 className="u-title-h2">{t('personalData.title')}</h2>
+        <h3 className="u-title-h3">{t('personalData.subtitle')}</h3>
```

Comment:
Why do we need to do that ? Aren't there typographic components in cozy-ui ?

Reply:
fixed it after @y-lohse 's comment, thanks